qemu/tests/hotplug_mem_repeat.py

```diff
@@ -35,12 +36,17 @@ def start_test(self):
         memory 256 times, then unplug 256 times. Otherwise, repeat hotplug
         and unplug in turn for 256 times. This is test entry.
         """
-        times = int(self.params["repeat_times"])
+        times = self.params.get_numeric("repeat_times", int)
+        get_basic_page = self.params.get("get_basic_page", False)
         target_mems = []
         for i in range(times):
             target_mems.append("mem%s" % i)
         vm = self.env.get_vm(self.params["main_vm"])
         session = vm.wait_for_login()
+        if get_basic_page:
+            self.test.log.info("Check basic page size on guest.")
+            if session.cmd(get_basic_page).strip() == '65536':
+                self.params['size_mem'] = self.params.get("size_mem_64k")
```

Comment:
How do we handle the `params['size_mem']` if not `session.cmd(get_basic_page).strip() == '65536'`?

Reply:
get default value from cfg